docs/topics/wasm/wasm-js-interop.md

```diff
@@ -395,10 +397,23 @@ external fun <T : JsAny> processData(data: JsArray<T>): T
 
 ## Exception handling
 
-You can use Kotlin `try-catch` expression to catch JavaScript exceptions.
-However, accessing specific details about the thrown value in Kotlin/Wasm isn't possible by default.
+You can use Kotlin `try-catch` expressions to catch JavaScript exceptions in Kotlin/Wasm code.
+The caught exceptions include detailed information:
+
+* When exceptions are thrown from JavaScript: you can see more detailed information on Kotlin's side.
+  When such an exception propagates through Kotlin back to JS, it's no longer wrapped into WebAssembly.
+* When exceptions are thrown from Kotlin: they can now be caught on JavaScript's side as JS errors.
+
+This exception handling works automatically in modern browsers
+that support the [`WebAssembly.JSTag`](https://webassembly.github.io/exception-handling/js-api/#dom-webassembly-jstag)
+feature:
+
+* Chrome 115+
+* Firefox 129+
+* Safari 18.4+
 
-You can configure the `JsException` type to include the original error message and stack trace from JavaScript.
+In older browsers, you have to configure the `JsException`
```

Comment:
Maybe it's worth putting the part for the older browsers into a collapsed block?